config/package/socorro/local.py

```diff
@@ -158,3 +158,45 @@
 # https://docs.djangoproject.com/en/dev/topics/http/\
 #  file-uploads/#changing-upload-handler-behavior
 #FILE_UPLOAD_PERMISSIONS = 0664
+
+DATASERVICE_CONFIG_BASE = {
+    'resource': {
+        'postgresql': {
+            'transaction_executor_class':
+                'socorro.database.transaction_executor'
+                '.TransactionExecutorWithLimitedBackoff',
+            'backoff_delays': "0, 3",
+        },
+    },
+    'secrets': {
+        'postgresql': {
+            'database_password': 'aPassword',
```

Comment:
```local.py``` will go away once we get the django app on consul (PR #2684), the ```breakpad_rw```/```aPassword``` has been the default for example configs so far so should work in the meantime.